packages/patterns-react/src/patterns/CardArray/CardArray.js

```diff
@@ -28,46 +30,38 @@ const CardArray = ({ title, content }) => {
   const containerRef = useRef();
 
   useEffect(() => {
-    setCardsHeight();
+    setSameHeight();
+    window.addEventListener('resize', setSameHeight);
   });
 
   /**
-   * Set the cards to have the same height as the bigger one
+   * Function that activates the sameheight utility
    */
-  const setCardsHeight = () => {
-    let biggest = 0;
-    const cards = Array.prototype.slice.call(containerRef.current.children);
-    cards.forEach(card => {
-      if (card.offsetHeight > biggest) {
-        biggest = card.offsetHeight;
-      }
-    });
-    cards.forEach(card => {
-      card.style.height = biggest + 'px';
-    });
+  const setSameHeight = () => {
+    sameheight(
+      containerRef.current.getElementsByClassName(
+        `${prefix}--card-link__title`
+      ),
+      'md'
+    );
+    sameheight(containerRef.current.children, 'md');
   };
 
   return featureFlag(
     DDS_CARD_ARRAY,
-    <section
-      className={`${prefix}--cardarray`}
-      data-autoid={`${stablePrefix}--cardarray`}>
-      <div className={`${prefix}--cardarray__container`}>
-        <div className={`${prefix}--cardarray__row`}>
-          <div className={`${prefix}--cardarray__col`}>
-            <h3 className={`${prefix}--cardarray__title`}>{title}</h3>
-          </div>
+    <section data-autoid={`${stablePrefix}--cardarray`}>
```

Comment:
Let's say we are going to use a wrapper to do this, I'm still not sure `section` would be the appropriate one to use. I almost feel like `article` would be better because this is a self-contained piece of content. I still kind of feel like that belongs in the `ContentGroup` though.